packages/account-v2/src/pages/ConnectedApps/ConnectedApps.tsx

```diff
@@ -10,12 +10,16 @@ import { ConnectedAppsSidebar } from './ConnectedAppsSidebar';
 import { ConnectedAppsTable } from './ConnectedAppsTable';
 
 export const ConnectedApps = () => {
-    const { data: connectedApps, isError, isLoading } = useFetchConnectedApps();
-    const { mutate: revokeMutate } = useRevokeConnectedApps();
+    const { data: connectedApps, isError: isFetchError, isLoading: isFetchLoading } = useFetchConnectedApps();
+    const { isError: isRevokeError, isLoading: isRevokeLoading, mutate: revokeMutate } = useRevokeConnectedApps();
+
     const { isMobile } = useDevice();
     const [isModalOpen, setIsModalOpen] = useState(false);
     const [selectedAppId, setSelectedAppId] = useState<number | null>(null);
 
+    const isLoading = isFetchLoading || isRevokeLoading;
+    const isError = isFetchError || isRevokeError;
+
     const handleToggleModal = useCallback((appId: number | null = null) => {
```

Comment:
non of the components which accepts handleToggleModal as a prop is wrapped  with memo()
so there is no sense to use useCallback here. it just adds additional logic